content/docs/userspace/dist/tools.md

```diff
@@ -0,0 +1,372 @@
+  %cz hash:         0v6.2nqmu.oqm24.ighl6.n0gp9.s8res.feql1.dl8ap.isli3.jk0hu.acrd2
+  app status:       running
+  force on:         ~
+  force off:        ~
+  publishing ship:  ~
+  updates:          tracking
+  source ship:      ~zod
+  source desk:      %base
+  source aeon:      3
+  pending updates:  ~
+::
+%garden
+  /sys/kelvin:      [%zuse 420]
+  base hash:        ~
+  %cz hash:         0v1e.2h7hs.elq3g.1sdt7.qfga6.ganga.7p95j.aog44.8p5fe.kpr6v.7ai82
+  app status:       running
+  force on:         ~
+  force off:        ~
+  publishing ship:  ~
+  updates:          tracking
+  source ship:      ~zod
+  source desk:      %garden
+  source aeon:      3
+  pending updates:  ~
+```
+
+---
+
+## `+agents`
+
+Print out the status of Gall agents on a desk.
+
+Agents may either be `archived` or `running`. Nuked or unstarted agents which are not on the manifest are omitted.
+
+#### Arguments
+
+```
+desk
+```
+
+#### Example
+
+```
+> +agents %landscape
+status: running   %notify
+status: running   %group-store
+status: running   %metadata-store
+status: running   %graph-push-hook
+status: running   %group-view
+status: running   %hark-graph-hook
+status: running   %hark-group-hook
+status: archived  %chat-cli
+status: running   %metadata-pull-hook
+status: running   %contact-pull-hook
+status: running   %contact-push-hook
+status: running   %invite-hook
+status: running   %contact-view
+status: running   %invite-view
+status: running   %s3-store
+status: running   %contact-store
+status: running   %clock
+status: running   %group-pull-hook
+status: running   %launch
+status: running   %metadata-push-hook
+status: running   %group-push-hook
+status: running   %graph-pull-hook
+status: running   %weather
+status: running   %dm-hook
+status: running   %sane
+status: running   %contact-hook
+status: running   %invite-store
+status: running   %graph-store
+status: running   %metadata-hook
+status: running   %chat-view
+status: running   %observe-hook
+status: running   %hark-invite-hook
+status: running   %chat-hook
+status: running   %hark-chat-hook
+status: running   %chat-store
+```
+
+---
+
+## `|suspend`
+
+Shut down all agents on a desk, archiving their states.
+
+The tile in the homescreen (if it has one) will turn gray and say "Suspended" in the top-left corner. This generator does the same thing as selecting "Suspend" from an app tile's hamburger menu.
+
+#### Arguments
+
+```
+desk
+```
+
+#### Examples
+
+```
+|suspend %bitcoin
+```
+
+---
+
+## `|revive`
+
+Revive all agents on a desk, migrating archived states.
+
+All agents specified in `desk.bill` which are suspended will be restarted. If updates to the agents have occurred since their states were archived, they'll be migrated with the state transition functions in the agent. This generator does the same thing as selecting "Resume App" from the app tile's hamburger menu.
+
+#### Arguments
+
+```
+desk
+```
+
+#### Examples
+
+```
+|revive %bitcoin
+```
+
+---
+
+## `|install`
```

Comment:
Worth noting that, as a parallel to `|uninstall` & `|suspend`, `|install` also does`|revive` behavior of the installed apps had previously been installed at some point.

Reply:
I tried and it seems it doesn't actually revive suspended agents